src/claim/entry/entry.js

```diff
@@ -60,6 +60,17 @@ class Entry {
   toHexadecimal() {
     return utils.bytesToHex(Buffer.concat(this._elements));
   }
+
+  /**
+   * String deserialization into entry element structure
+   * @param {String} Hexadecimal string representation of element claim structure
+   */
+  fromHexadecimal(entryHex) {
+    const entryBuff = utils.hexToBytes(entryHex);
+    for (let i = 0; i < this._elements.length; i++) {
+      this._elements[(this._elements.length - 1) - i] = entryBuff.slice(entryBuff.length - (32 * (i + 1)), entryBuff.length - 32 * i);
```

Comment:
I know that I'm kind of fool guy :) but please, could you comment for others fool developers what are you doing here? :) thanks! (why the 32, etc....)

Reply:
update comment inside the loop :)